index.html

```diff
@@ -0,0 +1,137 @@
+            <div class="home-content">
+              <div class="info-log">
+                <img src="./images/program/1 lecture.png" alt="Lecture" />
+                <h3>Lecture</h3>
+                <p>
+                  Listen to the speakers from various countries about the message of innovation and technology in Afrika.
+                </p>
+              </div>
+              <div class="info-log">
+                <img src="./images/program/1 exhibition.png" alt="IX Exhibition" />
+                <h3>IX Exhibition</h3>
+                <p>
+                  Appreciate various innovations applying the IX patent presented by innovators, hobbyists and hackers.
+                </p>
+              </div>
+              <div class="info-log">
+                <img src="./images/program/1 forum.png" alt="Forum" />
+                <h3>Forum</h3>
+                <p>
+                  Get the opportunity to share your thoughts and opinions with experts and thought leaders in tech industry.
+                </p>
+              </div>
+              <div class="info-log">
+                <img src="./images/program/1 workshop.png" alt="Workshop" />
+                <h3>Workshop</h3>
+                <p>
+                  Try creating your own work using the open source model instead of just being a spectator in the industry.
+                </p>
+              </div>
+              <div class="info-log">
+                <img src="./images/program/1 ignite.png" alt="IX Ignite" />
+                <h3>IX Ignite</h3>
+                <p>
+                  Get the opportunity to network with IX Summit attendees and supports across the continent even after the summit.
+                </p>
+              </div>
+            </div>
+            <div class="center-content">
+              <button class="join-btn">Join CS African Summit 2023</button>
+              <p class="main-footer">SEE THE WHOLE PROGRAM</p>
+            </div>
+          </section>
+          <section id="profiles" class="speaker-profiles">
+            <h3 class="talkers">Featured Speakers</h3>
+            <div class="guests"></div>
+          </section>
+          <section class="sponsors">
+            <h2>Partner</h2>
+            <div class="sponsor">
+              <img src="./images/partners/youtube.png" alt="YouTube" width="100px" />
+              <img src="./images/partners/facebook.png" alt="Meta" width="100px" />
+              <img src="./images/partners/windows.png" alt="Microsoft" width="100px" />
+              <img src="./images/partners/ford.png" alt="Ford Motors" width="100px" />
+              <img src="./images/partners/grand.png" alt="Ford Motors" width="100px" />
+            </div>
+          </section>
+          <section class="footer">
+            <div>
+              <img src="./images/logos/CSi.png" width="30px" />
+            </div>
+            <div>&copy 2023 - InnovationX Summit</div>
+          </section>
```

Comment:
- Hey, in here, this is the footer of your page which should be a `<footer>` tag and not a `<section>`.  As the footer is meant to be a standalone element in your webpage, you should move it out of this section. kindly update this to improve your semantic usage and to improve your page layout.